docs/concepts/storage-profiles-content.rst

```diff
@@ -0,0 +1,19 @@
+Flocker Storage Profiles are a way to help you choose different levels of service from your storage provider.
+For example, in a development environment you might only want a low cost storage option.
+However, if you need storage for a production environment, you can choose a profile with high performance and high cost options.
+
+Flocker Storage Profiles require support from your storage driver, and you are able to choose from the following profiles:
+
+* Gold: This profile is typically for high performance storage.
```

Comment:
Here as well - would be better to present this from application point of view. For example:
```Gold - This profile is suited for applications with high performance needs from storage (for example: databases).```

The reason for presenting ``Gold`` from an application point of view (rather than storage point of view) is as follows: consider an all-flash array storage provider. In this case, gold/silver/bronze would all be backed by high performance storage. What would differ for gold/silver/bronze in this case is IOPS shares, bandwidth, and other such knob settings.

Thanks!